src/libsync/creds/oauth.cpp

```diff
@@ -557,4 +549,70 @@ void OAuth::openBrowser()
     });
 }
 
+void OAuth::dynamicRegistrationDataReceived(const QVariantMap &dynamicRegistrationData)
+{
+    // the default class (used by the wizard only) doesn't know how to handle such data, and also doesn't need to
+    // only the derived AccountBasedOAuth class must be able to handle the data
+    (void)dynamicRegistrationData;
```

Comment:
Q_UNUSED? or unnamed parameter?
I guess dynamicRegistrationDataReceived could just be a signal?

Reply:
It used to be, when the factory (which then became a job, and later was replaced by a derived class) handled all these details, and the `OAuth` class did not have to know about such details. Now, there is no reason to expose this signal to the public any more.